### ISSUE TITLE
Unexpected banner `release-1.7/bf53498635 (fork: 461 commits, 252 days)`

### ISSUE THREAD

**Issue by ericphanson** [JuliaLang/juliaup#251]
```julia
❯ which julia
/Users/eph/.juliaup/bin/julia

❯ juliaup --version
Juliaup 1.5.36

❯ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.7.2 (2022-02-06)
 _/ |\__'_|_|_|\__'_|  |  release-1.7/bf53498635 (fork: 461 commits, 252 days)
|__/                   |

```
       
I expected the usual `Official https://julialang.org/ release` banner instead of `release-1.7/bf53498635 (fork: 461 commits, 252 days)`. E.g. when I download the binaries from https://julialang.org/downloads/ on the same computer,
```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.7.2 (2022-02-06)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |


```

**Comment by simeonschaub** [comment]
What's the exact platform?

**Comment by ericphanson** [comment]
```julia
julia> versioninfo()
Julia Version 1.7.2
Commit bf53498635 (2022-02-06 15:21 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.5.0)
  CPU: Apple M1
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, westmere)
Environment:
  JULIA_NUM_THREADS = 4
  JULIA_PKG_SERVER_REGISTRY_PREFERENCE = eager
```

m1 macbook pro, intel binaries (but same issue with the aarch64 binaries)

**Comment by davidanthoff** [comment]
@staticfloat is the version that is in the .tar.gz files somehow different than the version in the dmg files?

**Comment by staticfloat** [comment]
Sigh, it's built from the same git commit, but it must have been missing an environment variable or git tag, causing it to not print out the correct banner.

It would be nice to fix this, but it's a completely aesthetic (and non-functional) difference.

**Comment by KristofferC** [comment]
I think this is fixed now (1.7.3).
